src/recipes_jp.yaml

```diff
@@ -0,0 +1,359 @@
+# All cooking outputs must be listed here, except when cooking a Mystery
+Products:
```

Comment:
For awareness: similar to what happened to the actors yaml file, I had to create a separate recipes files for JP (only 3 recipes are different, though).